js/components/MagnetMapView.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Dimensions,
+  StyleSheet,
+} from 'react-native';
+import MapView from 'react-native-maps';
+
+export default class MagnetMapView extends Component {
+  render() {
+    const { height, width } = Dimensions.get('window');
```

Comment:
+1 to @gmarty ... I'm pretty sure we will be polishing the components as we use them.